enterprise/core-edge/src/test/java/org/neo4j/coreedge/raft/log/ConcurrentStressIT.java

```diff
@@ -141,14 +148,16 @@ private void write( RaftLog raftLog )
         }
     }
 
+    private static final CharSequence CHARS = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
```

Comment:
Windows is special as usual, but I think it would work if DummyRaftableContentSerializer were switched to use the the StringMarshal which marshals correctly for all characters, even on Windows. That is likely the main issue.